src/main/scala/org/broadinstitute/dsde/workbench/sam/service/UserService.scala

```diff
@@ -103,7 +107,9 @@ class UserService(val directoryDAO: DirectoryDAO, val cloudExtensions: CloudExte
     for {
       existingUser <- newWorkbenchUser match {
         case SamUser(_, Some(googleSubjectId), _, _, _, _) => directoryDAO.loadSubjectFromGoogleSubjectId(googleSubjectId, samRequestContext)
+          .withComputedDebugLogMessage(subject => s"New workbench user with id ${newWorkbenchUser.toUserIdInfo.userSubjectId} has Google subject: $subject")
```

Comment:
Not sure what we mean when we say `Google subject` here.  Maybe just:

```suggestion
          .withComputedDebugLogMessage(subject => s"New workbench user with id ${newWorkbenchUser.toUserIdInfo.userSubjectId} WorkbenchSubject: $subject")
```